jumpscale/sals/marketplace/apps_chatflow.py

```diff
@@ -40,6 +44,23 @@ def _init_solution(self):
         self.currency = "TFT"
         self.identity_name = j.core.identity.me.instance_name
 
+    @property
+    def branch(self):
+        if not self._branch:
+            try:
+                path = j.packages.threebot_deployer.__file__
+                git_client = j.clients.git.get("default_threebot")
+                git_client.path = j.sals.fs.parents(path)[3]
```

Comment:
not loving this but I understand it works

Reply:
will be replaced with this method: https://github.com/threefoldtech/js-ng/pull/548
upon js-ng release